lmp/bb-config.sh

```diff
@@ -163,6 +163,8 @@ if [ "${DISABLE_GPLV3}" = "1" ]; then
 INHERIT += "image-license-checker lmp-disable-gplv3"
 IMAGE_LICENSE_CHECKER_ROOTFS_BLACKLIST = "GPL-3.0 LGPL-3.0 AGPL-3.0"
 IMAGE_LICENSE_CHECKER_NON_ROOTFS_BLACKLIST = "GPL-3.0 LGPL-3.0 AGPL-3.0"
+IMAGE_LICENSE_CHECKER_ROOTFS_DENYLIST = "GPL-3.0* LGPL-3.0* AGPL-3.0*"
+IMAGE_LICENSE_CHECKER_NON_ROOTFS_DENYLIST = "GPL-3.0* LGPL-3.0* AGPL-3.0*"
```

Comment:
The new variables with `DENYLIST` is added but the old ones `BLACKLIST` is not deleted, shouldn't we erase the old ones?

Reply:
The ci-script at master is used on all factories, so we let the BLACKLIST for backport compatibility (for the factories using v86 and older)

Reply:
Understood, thanks